streams/mx.m3u

```diff
@@ -237,3 +235,13 @@ https://5ca9af4645e15.streamlock.net/zacatecas/videozacatecas/playlist.m3u8
 https://channel07-notusa.akamaized.net/hls/live/2036971-b/event01/index.m3u8
 #EXTINF:-1 tvg-id="",TELE SISTEMA CANAL 9 (486p) [Geo-blocked]
 http://k4.usastreams.com/ARBtv/ARBtv/playlist.m3u8
+#EXTINF:-1 tvg-id="TUDNMexico.mx",TUDN Mexico (540p)
+http://190.61.102.67:2000/play/a08j/index.m3u8
+#EXTINF:-1 tvg-id="Canal22Nacional.mx",Canal22Nacional.mx (720p) [Geo-blocked]
+http://mc-qs.lanesh4d0w.tech/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/canal22/
+#EXTINF:-1 tvg-id="",Las Estrellas (720p)
+https://channel01.akamaized.net/hls/live/2022749/event01/index.m3u8
```

Comment:
Seems to be the National stream (outside of the US citizens, could be having a timezone change)

Reply:
Could you provide a specific ID so I can add it to the playlist?

Reply:
Could be LasEstrellas.mx